### ISSUE TITLE
error when using with typescript 2.4.2

### ISSUE THREAD

**Issue by amir-arad** [fugazi-io/connector.node#5]
```
node_modules/@fugazi/connector/scripts/bin/common/map.d.ts(23,22): error TS2420: Class 'ExtendedMap<K, V>' incorrectly implements interface 'Map<K, V> & { empty(): boolean; keysArray(): K[]; valuesArray<V>(): V[]; toArray<T>(mapper: (valu...'.
  Type 'ExtendedMap<K, V>' is not assignable to type '{ empty(): boolean; keysArray(): K[]; valuesArray<V>(): V[]; toArray<T>(mapper: (value: V, key: K...'.
    Types of property 'valuesArray' are incompatible.
      Type '() => V[]' is not assignable to type '<V>() => V[]'.
        Type 'V[]' is not assignable to type 'V[]'. Two different types with this name exist, but they are unrelated.
          Type 'V' is not assignable to type 'V'. Two different types with this name exist, but they are unrelated.
```

**Comment by amir-arad** [comment]
seems to have been fixed in #4 
specificaly https://github.com/fugazi-io/connector.node/pull/4/files#diff-0f2fa6ded7dba7519a1726af82e5cc5aR98
all that is missing now is a version bump & release ro npmjs

**Comment by nitzantomer** [comment]
@amir-arad 
version number also updated to `1.0.15` and updates were published to npm
